tests/fixtures.py

```diff
@@ -92,29 +91,56 @@ def finalizer():
     assert class_instance.sc_obj, "Failed to create storage class"
 
 
+@pytest.fixture()
+def create_cephfs_storageclass(request):
+    """
+    Create a CephFS storage class
+    """
+    class_instance = request.node.cls
+
+    def finalizer():
+        """
+        Delete the CephFS storage class
+        """
+        if class_instance.sc_obj.get():
+            class_instance.sc_obj.delete()
+
+    request.addfinalizer(finalizer)
+
+    class_instance.sc_obj = helpers.create_storage_class(
+        interface_type=constants.CEPHFILESYSTEM,
+        interface_name=helpers.get_cephfs_data_pool_name(),
+        secret_name=class_instance.cephfs_secret_obj.name
+    )
+    assert class_instance.sc_obj, f"Failed to create storage class"
+
+
 @pytest.fixture()
 def create_pvc(request):
     """
     Create a persistent Volume Claim
     """
     class_instance = request.node.cls
 
+    def finalizer():
+        """
+        Delete the PVC
+        """
+        class_instance.pvc_obj.delete()
+
+    request.addfinalizer(finalizer)
     class_instance.pvc_obj = helpers.create_pvc(
         sc_name=class_instance.sc_obj.name
     )
 
 
 @pytest.fixture()
-def create_pod(request):
+def create_rbd_pod(request):
     """
     Create a pod
     """
     class_instance = request.node.cls
 
```

Comment:
finalizer()?

Reply:
No, there are tests that use this fixture which are creating the pod during the test itself so there is a need to separate the teardown finalizer from the setup fixture here.
For further details, see the discussion here: https://github.com/red-hat-storage/ocs-ci/pull/339#issuecomment-509931850